packages/framework/src/client.ts

```diff
@@ -384,6 +380,14 @@ export class Client {
     };
   }
 
+  private shouldSanitize({ stepType, options }: { stepType: StepType; options: ChannelStepOption | undefined }) {
```

Comment:
at the moment, we implemented more precise sanitization, applying it only to email and in-app. because for other channels, there's no reason to sanitize, which reduces computation and complexity for framework users.

Reply:
Nit-picky one: You can keep it simple for now and do `(stepType: string, disableOutputSanitization: boolean);` to avoid the extra types and such.

Reply:
in that case, we will have another compilation error. 
i believe this one is the [issue](https://linear.app/novu/issue/NV-5508/fix-return-value-of-executestepfactory-in-novuframework).